group_vars/all/eda_credentials.yml

```diff
@@ -2,7 +2,9 @@
 eda_credentials:
   - name: hub
     description:  private automation hub
+    organization: config_as_code
```

Comment:
this looks good, but talking with the engineers, eda doesn't use these at all, so I'd just remove the entry, AAP credentials in general.